src/rendering/RenderUtil.cc

```diff
@@ -92,6 +92,10 @@
 #include "ignition/gazebo/rendering/SceneManager.hh"
 #include "ignition/gazebo/rendering/MarkerManager.hh"
 
+#include <ignition/gui/Application.hh>
+#include <ignition/gui/MainWindow.hh>
+#include "ignition/gazebo/gui/GuiEvents.hh"
```

Comment:
We shouldn't add GUI as a dependency of rendering because `RenderUtil` is also used by the backend in `systems::Sensors`. See a suggestion in https://github.com/ignitionrobotics/ign-gazebo/pull/1178